corebehrt/functional/files_and_folders.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+
+def split_path(path_str: str) -> list:
```

Comment:
This can be simplified using `pathlib Path` by doing `Path(path_str).parts` to split up and extract directories (by doing `parts[:-1]`

Reply:
Addressed in 52be7d6314f3ad08240774c9eefd34fa17927768